src/index.html

```diff
@@ -22,6 +22,442 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </div>
+      <nav class="header__nav">
+        <a
+          href="./index.html"
+          class="header__link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Samsung
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Smartphones
+        </a>
+        <a
+          href="./index.html"
+          data-qa="nav-hover"
+          class="header__link"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Gadgets
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Tablets
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Photo
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Video
+        </a>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <card
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__header">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__main">
+          <h1 class="card__product-name">
+            APPLE A1419 iMac 27&#8220; Retina 5K Monoblock (MNED2UA&#47;A)
+          </h1>
+
+          <h2 class="card__product-code">Product code: 195434</h2>
+
+          <container class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+
+            <div class="stars__reviews">Reviews: 5</div>
+          </container>
+
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+
+            <div class="card__price-value">$2,199</div>
+          </div>
+        </div>
+
+        <div class="card__button">
+          <a
+            href="#"
+            data-qa="card-hover"
+            class="card__button-link"
+          >
+            buy
+          </a>
+        </div>
+      </card>
+
+      <card
```

Comment:
Such a tag does not exist, so it must be fixed everywhere
```suggestion
      <article
```